src/tutorials/0008-introduction-to-libp2p/04.md

```diff
@@ -0,0 +1,21 @@
+---
+    title: "libp2p in the OSI Model"
+    type: "multiple-choice"
+---
+
+So where does libp2p fit into the current web?
+Usually, we look at the **OSI Model (Open Systems Interconnection Model)** to understand which layer a protocol tries to tackle.
```

Comment:
Again, nothing in this diagram is familiar to a "normal" person except maybe HTTP and ethernet. We need to either make this content accessible to everyone (with much less assumed knowledge) or put disclaimers on it so only people who already understand networking will explore this tutorial.

Reply:
Again, I didn't intend for the reader to understand everything, just that there are basic building blocks when designing network systems, and that OSI Models are usually the go to models to try and fit protocols into.

We can either remove this section, or add the note that the content of the diagram is not really important, but rather the notion of the complexity of the system.

Reply:
How about preceding the OSI sentence with something like "The web is built on a complex system of networking protocols, some of which are familiar to end users, like HTTP and ethernet, and some of which are only familiar to networking specialists."